src/Sarif/Schemata/sarif-schema.json

```diff
@@ -1389,6 +1394,47 @@
             }
         },
 
+        "reportingDescriptorTranslation": {
+            "description": "tbd",
+            "additionalProperties": false,
+            "type": "object",
+            "properties": {
+
+                "id": {
+                    "description": "TBD matches reportingDescriptor.id",
+                    "type": "string"
+                },
+
+                "guid": {
```

Comment:
>guid [](start = 17, length = 4)

"The unique identifier in the form of a GUID of the reporting descriptor to which this translation belongs, matching reportingDescriptor.guid."